src/StackAdmin/Azs.Commerce.Admin/Module/Azs.Commerce.Admin/0.1.0/Test-CoreRequirements.ps1

```diff
@@ -0,0 +1,25 @@
+<#
+The MIT License (MIT)
```

Comment:
What is this file for?

Reply:
The PSSwagger team use it to test where the DLLs are located so they can load them for the modules.